src/essential_validator.py

```diff
@@ -392,38 +392,42 @@ def check_relationship(self, file_info, type, columns):
                 return True, None
             else: # check if invalid relationships
                 msg = f'Relationships {json.dumps(rel_props)} are not defined.' if len(rel_props) > 1 else f'Relationship "{rel_props[0]}" is not defined.'
```

Comment:
I've never seen this message, will it still happen? What's the situation cause this error?

Reply:
A tsv file with invalid relationship(s) while no parents defined in data model.   e.g.  add relationship(s) in a tsv file for program node type, you eill see the message.

Reply:
In that case, let's change the message to "**Relationship column "test.test_id" is invalid, "program" node should not have any relationship columns.**". Also, please split it into multiple errors if there are multiple relationship columns.

Reply:
updated